### DISCUSSION TITLE
Underline OSC 8 Anchor text hyperlinks

### DISCUSSION THREAD

**Discussion by HaleTom** [wezterm/wezterm#4057] [Q&A] [answered]
[Wezterm and most recent VTE-based terminal emulators support hyperlinks via an escape code](https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda).

I can't find the relevant VTE documentation, but iTerm's documentation for the [Anchor OSC 8 escape code](https://iterm2.com/documentation-escape-codes.html#:~:text=first%20two%20digits.-,Anchor%20(OSC%208),-VTE%20and%20iTerm2) allows anchor text with links to files which include line numbers.

An example of an Anchor escape code is:

    printf '\e]8;;http://example.com\e\\This is a link\e]8;;\e\\\n'

I have already got the config line:
```
config.hyperlink_rules = wezterm.default_hyperlink_rules()
```

How do I get a underline on the printed text `This is a link` in the example above?

**Comment by HaleTom** [comment]
Previously asked with only downvotes as a reply at [Unix & Linux StackExchange](https://unix.stackexchange.com/questions/752130/highlight-osc-8-anchor-text-hyperlinks-in-wezterm)

**Comment by wez** [comment, marked as answer]
wezterm shows the underline on a hyperlink only when you hover

**Reply by HaleTom** [reply]
Cheers, I raised this as an improvement: https://github.com/wez/wezterm/issues/4077
